QUEST.tsv

```diff
@@ -2098,156 +2098,156 @@ QUEST_20150714_002097	But, Raius is the disciple I acknowledge.{nl}His passion t
-QUEST_20150717_002250	Please insert the name of your companion.
+QUEST_20150714_002101	$Hones, who is one of them, is lacking in scholarly intelligence. However, I am sure his faithfulness and efforts will help him follow my teachings.
+QUEST_20150714_002102	$The Records of the Hidden Object
+QUEST_20150714_002103	$…I don't know whether or not I should thank the Goddesses for this discovery. My greed for learning can ruin the world.
+QUEST_20150714_002104	$I can't sleep well for fear of my research results being spread by demon investigation. {nl}That's why I wish to pass the rest of the research to the one I can trust.
+QUEST_20150714_002105	$The Inheritance of the Master
+QUEST_20150714_002106	$As the disciples and theologians know, Goddess Bibora is in charge of the library where all records of the world are stored. {nl}I was able to check several records in that fantastic library after the research and long studying I can't tell anyone about at the moment. {nl}
+QUEST_20150714_002107	$From the many records of the Goddess, I was only able to see a few. {nl}Nevertheless, the fact that I checked the existence of the records with the important facts was astonishing. {nl}
+QUEST_20150714_002108	$The record which I can use to find the whereabouts of all the Goddesses… {nl}If you could obtain that secret information, we might be able to predict the Goddesses' future locations by analyzing what they did in the past. {nl}Because of that, your future plans may be disturbed or put at risk. {nl}
+QUEST_20150714_002109	$This is more dangerous because, if the demons find out the truth, their greed towards the library will grow, {nl}and if that greed is fulfilled, the world will be swallowed by evil. That's why…
+QUEST_20150714_002110	$(The rest is blurred and unreadable.)
+QUEST_20150714_002111	$The Memo of Disciple Hones
+QUEST_20150714_002112	$If you want to find the master's research, you have to find three spatial magic stones and activate the fusion device.
+QUEST_20150714_002113	$The spatial magic stone of the contract is at the reception room. {nl}The monsters probably have the fragments. {nl}Collect all the fragments of the magical stones and go to the preserved magic field in the Grave of the Saints to restore them.
+QUEST_20150714_002114	$The spatial magic stones of sacrifices are with the Templeshooter Witches. {nl}The guiding stone of wisdom will help you to find the hidden Templeshooter.
+QUEST_20150714_002115	$The spatial magic stones of the promise can be found in Siaura Mine. {nl}Use the essence of the spirit to the monsters first to collect their condensed resentment. {nl}With that, destroy the stones in Siaura Mine. {nl}Black chasers will pursue from behind, avoid them and go to the end of the mine to find the spatial magic stones of the promise.
```

Comment:
No need for "the" in "spatial magic stone of the promise". Also, singular again. This is the third spatial magic stone.